### PR TITLE
Serialize amount, currency, and time info for saving to metadata

### PR DESCRIPTION
**Pull request by jtormey**
We need this data to show the bitcoin received modal.

### CODE REVIEW

**Comment by Sjors** [comment]
I'm trying to store as little as possible in the metadata, to keep it as simple as possible. Can you instead, fetch the trade from the Coinify API before opening the modal? Or is that too slow?

File: src/coinify/trade.js [hunk 1]
```diff
@@ -133,6 +133,22 @@ Object.defineProperties(CoinifyTrade.prototype, {
   }
 });
 
+function toCents (fiat) {
```
Comment (Sjors) [inline src/coinify/trade.js]: These functions would be useful in other places too, so let's put them in coinify/helpers.js

File: src/coinify/trade.js [hunk 2]
```diff
@@ -149,27 +165,36 @@ CoinifyTrade.prototype.set = function (obj) {
...
+
+  if (obj.transferIn) {
+    this._medium = obj.transferIn.medium;
+    this._sendAmount = this._inCurrency === 'BTC'
+      ? toSatoshi(obj.transferIn.sendAmount)
+      : toCents(obj.transferIn.sendAmount);
+  }
+
+  if (this._inCurrency === 'BTC') {
+    this._inAmount = toSatoshi(obj.inAmount);
```
Comment (Sjors) [inline src/coinify/trade.js]: This change is useful even if we don't add more fields to the metadata.

File: src/coinify/trade.js (hunk 1 above)
Comment (jtormey) [inline src/coinify/trade.js]: Moved to helpers 👍

**Comment by jtormey** [comment]
Changes requested by @Sjors have been made

**Pull request merged**